src/services/adapters/chat-guidance-adapter/chat.guidance.event.type.ts

```diff
@@ -0,0 +1,5 @@
+export enum ChatGuidanceEventType {
```

Comment:
All enums under enum folder?

Reply:
It doesn't make sense to be in shared, it is used and will be used in only one place, so moved it there and made it private.